mc-sema/cfgToLLVM/raiseX86.h

```diff
@@ -358,7 +358,7 @@ void R_WRITE(llvm::BasicBlock *b, unsigned reg, llvm::Value *write) {
     throw TErr(__LINE__, __FILE__, "Could not find register");
 
   int regwidth = x86_64::REG_SIZE;
-  if (reg >= llvm::X86::XMM0 && reg <= llvm::X86::XMM7) {
+  if (reg >= llvm::X86::XMM0 && reg <= llvm::X86::XMM15) {
```

Comment:
Added a register width function that checks for unsupported registers. We don't have support for anything greater than XMM15 in `struct RegState`.